packages/contracts/tasks/deposits.ts

```diff
@@ -0,0 +1,54 @@
+import { task, types } from 'hardhat/config'
+import { Contract, providers, utils, Wallet } from 'ethers'
+import dotenv from 'dotenv'
+
+import * as depositFeedArtifact from '../artifacts/contracts/L1/DepositFeed.sol/DepositFeed.json'
+
+task('deposit', 'Deposits funds unto Optimism bedrock.')
+  .addParam('to', 'Recipient address.', null, types.string)
+  .addParam('amountEth', 'Amount in ETH to send.', null, types.string)
+  .addOptionalParam(
+    'depositContractAddr',
+    'Address of deposit contract.',
+    'deaddeaddeaddeaddeaddeaddeaddeaddead0001',
+    types.string
+  )
+  .setAction(async ({ to, amountEth, depositContractAddr }) => {
+    dotenv.config()
+
+    if (
+      !process.env.L1_PROVIDER_URL ||
```

Comment:
nit: having 2 ways to config, these should be default values in hardhat params so that there is a unified CLI experience